ietf/doc/views_doc.py

```diff
@@ -114,12 +114,12 @@ def render_document_top(request, doc, tab, name):
                 rsab_ballot,  
                 None if rsab_ballot else "RSAB Evaluation Ballot has not been created yet"
             ))
-    if doc.type_id in ("draft","conflrev", "statchg"):
-        tabs.append(("IESG Evaluation Record", "ballot", urlreverse("ietf.doc.views_doc.document_ballot", kwargs=dict(name=name)), iesg_ballot,  None if iesg_ballot else "IESG Evaluation Ballot has not been created yet"))
```

Comment:
This makes it so that the tab doesn't get placed there's no ballot. Earlier feedback was that people got lost if the tabs shifted around depending on document _state_. (Changing what tabs appear based on document _type_ is different).

Will it not cause issues to be raised when a chair notices that there is no IESG evaluation tab even through they've sent the document to the IESG? The current view pushes against that by explaining why the tab doesn't have content...

Reply:
Yeah, good point. I'll need to think about what condition would be correct...